roles/edpm_telemetry/defaults/main.yml

```diff
@@ -16,3 +16,9 @@
 
 
 # All variables intended for modification should be placed in this file.
+# Directory in the ansibleEE container
+edpm_ceilometer_config_src: /var/lib/openstack/config
```

Comment:
this would need to default to /var/lib/openstack/config/telemetry
The files mounted into the openstackansibleeee pod are put under /var/lib/openstack/<serviceName>:

https://github.com/openstack-k8s-operators/dataplane-operator/blob/b3826566d8a9dc881eb83a71fb59f096e5855145/pkg/deployment/deployment.go#L232

Reply:
Thank you!